shared/fs/common/path-item-action/menu.js

```diff
@@ -57,6 +59,30 @@ const hideMenuOnClick = (onClick: (evt?: SyntheticEvent<>) => void, hideMenu: ()
 
 const makeMenuItems = (props: Props, hideMenu: () => void) => [
   'Divider',
+  ...(props.newFolder
+    ? [
+        {
+          onClick: hideMenuOnClick(props.newFolder, hideMenu),
+          title: 'New folder',
+        },
+      ]
+    : []),
+  ...(props.openChatTeam
+    ? [
+        {
+          onClick: hideMenuOnClick(props.openChatTeam, hideMenu),
+          title: 'Chat with users in this team',
```

Comment:
I feel like "chat with the users in this team" more accurately describes this action to me and I'd prefer to add the word "the" there unless this is particularly space constrained.

Reply:
I tried adding "the" but it overflew into two lines. So not sure. Would you mind if I merge without the added "the", and discuss with #design?